tests/qos/test_buffer.py

```diff
@@ -2013,7 +2032,11 @@ def mellanox_calculate_headroom_data(duthost, port_to_test):
     if port_speed in pause_quanta_per_speed_dict.keys():
         pause_quanta = pause_quanta_per_speed_dict[port_speed]
     else:
-        pause_quanta = 60
+        use_default_peer_response_time = True
+        # Get default peer response time from State DB
+        # Command: redis-cli -n 6 hget "ASIC_TABLE|MELLANOX-SPECTRUM-3" "peer_response_time"
+        peer_response_time_keys = duthost.shell('redis-cli -n 6 keys ASIC_TABLE*')['stdout']
+        peer_response_time = float(duthost.shell('redis-cli -n 6 hget "{}" "peer_response_time"'.format(peer_response_time_keys))['stdout'])
```

Comment:
peer_response_time = value * 1024?

Reply:
not needed here, see line 2133:

`propagation_delay = port_mtu + bytes_on_cable + 2 * bytes_on_gearbox + MAC_PHY_DELAY + peer_response_time * 1024`